wagtail_link_block/templates/blocks/link_block.html

```diff
@@ -0,0 +1 @@
+<a href="{{ self.link.get_url }}" {% if self.link.new_window %}target="_blank"{% endif %}>{{ self.text }}</a>
```

Comment:
Currently we don't ship a template by default - which makes it a pain for testing!

I'll improve this in other PRs, so this is just an initial template.